feature/platform/fabric/otg_tests/fabric_redundancy_test/README.md

```diff
@@ -0,0 +1,72 @@
+# gNMI-1.14 Fabric redundnacy test
+
+## Summary
+- collect inventory data for each fabric card
+- Verify last restart time is updated
+- verify trassic could be forwarded with only one Fabric Card active.
```

Comment:
trassic -> traffic
But for this statement - why do we need to verify that? Probability of all but one fabrics going done is miserable. At the same time in prod we won't ever get to this state. We will more likely drain a device in case of a FC failure to get it back to the full scale. 
So, what's the goal of 1xFC test ?

Reply:
If we have last fabric standing and able communicate through it, i proves a) device can operate with failed fabrics. and b) internal handling of packets do not requite at lest N fabric in operation.
Alternativly we can try with failed one fabric. but this will not prove b)

On drain not - it takes time to drain and somthimes there is some ftaffic on device anyway. so between failure of fabric and drein be fully enforced, we relay of redundancy.